cfme/tests/configure/test_timeprofile.py

```diff
@@ -59,8 +61,12 @@ def test_days_required_error_validation():
         timezone="(GMT-10:00) Hawaii",
         days=False,
         hours=True)
-    with error.expected("At least one Day must be selected"):
-        tp.create()
+    if version.current_version() > "5.7":
+        tp.create(negative=True)
+        assert form_buttons.add.is_dimmed
```

Comment:
@psav , @mshriver We can't save form because if something invalid in 5.7+ "Add" button is bot clickable. So I add another verification. And yes, it is for all cases below